x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_pipeline.tsx

```diff
@@ -209,12 +212,21 @@ export const ConfigurePipeline: React.FC = () => {
                 )}
                 helpText={
                   !nameError &&
-                  i18n.translate(
-                    'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.name.helpText',
-                    {
-                      defaultMessage:
-                        'Pipeline names are unique within a deployment and can only contain letters, numbers, underscores, and hyphens.',
-                    }
+                  configuration.existingPipeline === false && (
```

Comment:
It looks like `existingPipeline` should already be a boolean. Any reason to explicitly check `false` vs `!existingPipeline`? I see we do the same thing elsewhere.

Reply:
when initialized `existingPipeline` is `undefined` which is when the select is on `Choose`. So this only removes the help text if you have existing pipeline choosen. I mainly did this to try and reduce layout shifts when selecting `New`

Reply:
Sounds reasonable enough 👍